base-image/Dockerfile

```diff
@@ -12,26 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Set up sshd. All this can go away once we stop using sshd.
 FROM alpine:3.13 as tel2-base
 RUN \
-  apk add --no-cache openssh ca-certificates && \
-  ssh-keygen -A && \
-  sed -i \
-    -e 's/^AllowTcpForwarding no/AllowTcpForwarding yes/' \
-    -e 's/^GatewayPorts no/GatewayPorts yes/' \
-    -e '$aClientAliveInterval 1' \
-    -e '$aPermitEmptyPasswords yes' \
-    -e '$aPort 8022' \
-    -e '$aClientAliveCountMax 10' \
-    -e '$aPermitRootLogin yes' \
-    /etc/ssh/sshd_config && \
-  chmod -R g+r /etc/ssh && \
+  apk add --no-cache openssh-sftp-server && \
```

Comment:
Don't remove `ca-certificates`, we still need it for validating Ambassador Cloud's TLS cert.